### ISSUE TITLE
Wiki Feedback

### ISSUE THREAD

**Issue by niartenyaw** [imoran/alrightCupid#1]
# Design Docs Feedback

Great job! Let me know if you have any questions about any of my comments.
Remember that your wiki must be completed before you will be allowed to
start coding tomorrow.

## MVP List

- [x] Great! Could you add the other three required MVPs and a timeline
  next to each MVP?

## Wireframes & Component Hierarchy

- [x] The messaging view is missing. What do I see when sending messages
  to other people?
- [x] For the wireframes, can you label the components that are on the
  page?
- [x] For component hierarchy, each of the `IndexContainer`s will probably
  have a `Index` presentational component. That `Index` component will
  then render each `IndexItem`.

## State Shape

- [x] Great job! Let's talk about some pros/cons of organizing `answers`
  and `responses` like this in state

### Frontend

- [x] What frontend route will be used to see messages?
- [x] For the `current user's profile`, could you also use the route
  `/users/:userId`?

### API Endpoints

- [x] Will need `/api/session` routes!
- [x] How will you be recording responses to questions? Will probably need
  some `responses` endpoints

## Schema

- [x] For location, might want to think about different ways to store
  this. A string of the location `San Francisco, CA` is one way, but we
  need to determine distances between people. One way to do this is store
  a GPS location instead. There is a [`geocoder`](https://github.com/alexreisner/geocoder)
  gem that can help with this. Let me know if you'd like to take a look
  through this together.
- [x] The `Question_Responses` could use a foreign key reference to the
  `answers` table instead of its own `response` string, which would be a
  duplicate of the answer's `title`.
